src/modules/CRUD/components/Details/Read/AttachmentView/AttachmentList/AttachmentList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Attachment from '../Attachment';
+import Picture from '../Picture';
+
+import './styles.scss';
+
+const AttachmentList = ({
+  name,
+  attachments,
+}) => {
+  const Component = name === 'attachments' ? Attachment : Picture;
+  return (
+    <ul className="attachment-list">
```

Comment:
```suggestion
const AttachmentList = ({
  name,
  attachments,
  className = '',
  ...props
}) => {
  const Component = name === 'attachments' ? Attachment : Picture;
  return (
    <ul className={`attachment-list ${className}`} {...props}>
```

ou encore :

```suggestion
const AttachmentList = ({
  name,
  attachments,
  className = '',
  ...props
}) => {
  const Component = name === 'attachments' ? Attachment : Picture;
  return (
    <ul className={classnames('attachment-list', className)} {...props}>
```

Reply:
I don't undestand this suggestion, this ul tag doesn't need a dynamic className

Reply:
It is not about a need, but about a global philosophy (best practice) of creating component:
Whenever you use any (potentially external) component that will generate a DOM node, you may want to add any html attribute. At least for debugging purpose.

Forwarding unused non-specific props on sub-element is a way to preserve these expectations, and keep a clean component extendability & reusability without breaking it's own behaviors.

This allows kind of:
```jsx
<AttachmentList style={{ border: '1px solid red' }} />
<AttachmentList style={{ listStyleType: 'none' }} />
<AttachmentList onClick={() => console.log('I\'m here')} />
<AttachmentList className={isInViewport ? 'in-viewport' : 'out-of-sight'} />
<AttachmentList id="myCustomId" />
<AttachmentList id="htmlAnchor1" />
```


@jrmi any thought on this ?